playtron-os-scripts/playtron-os-scripts.spec

```diff
@@ -15,23 +15,30 @@ Requires: cloud-utils-growpart fio
 %setup -q -c
 
 %install
-mkdir -p %{buildroot}/etc/sysctl.d/
-cp -Rv playtron-os-scripts-%{version}/etc/sysctl.d/* %{buildroot}/etc/sysctl.d/
-mkdir -p %{buildroot}/usr/bin/
-cp -Rv playtron-os-scripts-%{version}/bin/* %{buildroot}/usr/bin/
-mkdir -p %{buildroot}/usr/lib/systemd/system/
-cp -Rv playtron-os-scripts-%{version}/lib/systemd/system/* %{buildroot}/usr/lib/systemd/system/
+cp -Rv playtron-os-scripts-%{version}/etc/ %{buildroot}/
+
+mkdir -p %{buildroot}/usr/
+cp -Rv playtron-os-scripts-%{version}/bin/ %{buildroot}/usr/
+cp -Rv playtron-os-scripts-%{version}/lib/ %{buildroot}/usr/
+cp -Rv playtron-os-scripts-%{version}/share/ %{buildroot}/usr/
+
 mkdir -p %{buildroot}/usr/share/licenses/playtron-os-scripts/
-cp playtron-os-scripts-%{version}/LICENSE %{buildroot}/usr/share/licenses/playtron-os-scripts/LICENSE
+cp playtron-os-scripts-%{version}/LICENSE %{buildroot}/usr/share/licenses/playtron-os-scripts/
 
 %files
-/etc/sysctl.d/50-swappiness.conf
+/etc/gai.conf
```

Comment:
The `glibc` package provides this file so I was worried what would happen. Thankfully, it is a weak configuration file dependency and gets overridden by the `playtron-os-scripts` package. No action needed.

```
$ cat /etc/gai.conf
precedence ::ffff:0:0/96  100
$ sudo dnf install ~/rpmbuild/RPMS/noarch/playtron-os-scripts-0.4.0-1.fc38.noarch.rpm 
warning: /etc/gai.conf saved as /etc/gai.conf.rpmsave
$ cat /etc/gai.conf
label  ::1/128       0
label  ::/0          1
label  2002::/16     2
label ::/96          3
label ::ffff:0:0/96  4
precedence  ::1/128       50
precedence  ::/0          40
precedence  2002::/16     30
precedence ::/96          20
precedence ::ffff:0:0/96  100
$ cat /etc/gai.conf.rpmsave 
precedence ::ffff:0:0/96  100
```